src/index.ts

```diff
@@ -33,14 +39,26 @@ export function runInContext(
   context.errors = []
   const program = parse(code, context)
   if (program) {
-    const it = evaluate(program, context)
-    let scheduler: Scheduler
-    if (theOptions.scheduler === 'async') {
-      scheduler = new AsyncScheduler()
+    if (theOptions.isNativeRunnable) {
```

Comment:
Is it better to use a condition here or should we have a separate `runInNativeContext` or `nativeRunInContext` function instead? Not sure what the plans are for interoperability.

Reply:
I'd wait to see how debug and environment model works first.

Reply:
Roger. For now let's shelve the issue until we get more info.